lib/aot/compiler.cpp

```diff
@@ -163,12 +159,7 @@ struct SSVM::AOT::Compiler::CompileContext {
         Trap(llvm::Function::Create(
             llvm::FunctionType::get(VoidTy, {Int32Ty}, false),
             llvm::Function::PrivateLinkage, "trap", LLModule)),
-        Likely(llvm::MDTuple::getDistinct(
-            LLContext, {llvm::MDString::get(LLContext, "branch_weights"),
-                        llvm::ConstantAsMetadata::get(llvm::ConstantInt::get(
-                            LLContext, llvm::APInt(32, 2000))),
-                        llvm::ConstantAsMetadata::get(llvm::ConstantInt::get(
-                            LLContext, llvm::APInt(32, 0)))})) {
+        Likely(llvm::MDBuilder(LLContext).createBranchWeights(2000, 0)) {
```

Comment:
Why we need to set 2000 here? Is this a magic number?

Reply:
It's a magic number at https://github.com/llvm/llvm-project/blob/991e86156c88dfa64c908fe36da3a1945c3d2e50/llvm/lib/Transforms/Scalar/LowerExpectIntrinsic.cpp#L50
Looks like llvm have expect Intrinsic now, I'll change this to an intrinsic call.